components/menu/package.json

```diff
@@ -1,24 +1,16 @@
 {
-  "//": [
-    "================================================================================",
-    "# To work within the development environment, run the following tasks",
-    "  1. $ npm run dev",
-    "  2. Go to http://localhost:8000",
-    "================================================================================"
-  ],
-  "name": "@aurodesignsystem/auro-menu",
+  "name": "@auro-formkit/auro-menu",
   "version": "4.1.4",
   "description": "auro-menu HTML custom element",
+  "type": "module",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/AlaskaAirlines/auro-menu"
   },
-  "type": "module",
-  "main": "index.js",
-  "license": "Apache-2.0",
-  "engines": {
-    "node": "^18.x || ^20.x"
-  },
   "dependencies": {
     "@alaskaairux/icons": "^4.44.1",
     "@aurodesignsystem/auro-icon": "^6.0.2",
```

Comment:
@aurodesignsystem/auro-library version in this package.json is not aligned with the root's package.json

Reply:
Moved `auro-library` dependency to root `package.json` and removed from component packages.